lib/uniffi-zcash/tests/kotlin/zcash_primitives/legacy/keys/account_pub_key.kts

```diff
@@ -0,0 +1,72 @@
+import uniffi.zcash.*
+
+val supp = TestSupport.fromCsvFile()
+
+fun testAccountPubKeyNew() {
+	val ppkBytes = supp.getAsU8Array("account_public_key")
+
+	val ppk = ZcashAccountPubKey(ppkBytes)
+
+	assert(ppk.serialize() == ppkBytes)
+}
+testAccountPubKeyNew()
+
+fun testAccountPubKeyExternalIvk() {
+	val ppkBytes = supp.getAsU8Array("account_public_key")
+	val ivkBytes = supp.getAsU8Array("ppk_external_ivk")
+
+	val ppk = ZcashAccountPubKey(ppkBytes)
+	val ivk = ppk.deriveExternalIvk()
+
+	assert(ivk.toBytes() == ivkBytes)
+}
+testAccountPubKeyExternalIvk()
+
+fun testAccountPubKeyInternalIvk() {
+	val ppkBytes = supp.getAsU8Array("account_public_key")
+	val ivkBytes = supp.getAsU8Array("ppk_internal_ivk")
+
+	val ppk = ZcashAccountPubKey(ppkBytes)
+	val ivk = ppk.deriveInternalIvk()
+
+	assert(ivk.toBytes() == ivkBytes)
+}
+testAccountPubKeyInternalIvk()
+
+fun testAccountPubKeyForShielding() {
+	val ppkBytes = supp.getAsU8Array("account_public_key")
+	val intOvkBytes = supp.getAsU8Array("ppk_internal_ovk")
+	val extOvkBytes = supp.getAsU8Array("ppk_external_ovk")
+
+	val ppk = ZcashAccountPubKey(ppkBytes)
+	val ovks = ppk.ovksForShielding()
+
+	assert(ovks.internalOvk.asBytes() == intOvkBytes)
+	assert(ovks.externalOvk.asBytes() == extOvkBytes)
+}
+testAccountPubKeyForShielding()
+
+fun testAccountPubKeyInternalOvk() {
+	val ppkBytes = supp.getAsU8Array("account_public_key")
+	val ovkBytes = supp.getAsU8Array("ppk_internal_ovk")
+	val ppk = ZcashAccountPubKey(ppkBytes)
+	val ovk = ppk.internalOvk()
+
+	assert(ovk.asBytes() == ovkBytes)
+}
+testAccountPubKeyInternalOvk()
+
+fun testAccountPubKeyExternalOvk() {
+	val ppkBytes = supp.getAsU8Array("account_public_key")
+	val ovkBytes = supp.getAsU8Array("ppk_external_ovk")
+	val ppk = ZcashAccountPubKey(ppkBytes)
+	val ovk = ppk.externalOvk()
+
+	assert(ovk.asBytes() == ovkBytes)
+}
+testAccountPubKeyExternalOvk()
+
+fun testAccountPubKeySerialize() {
+    // covered by testAccountPubKeyNew()
+}
+testAccountPubKeySerialize()
```

Comment:
Should we remove this then ?

Reply:
There are other functions like this - I made and left them as IMO its easier to track if a method is already tested or not.
Bu I can remove them - what do you think?

Reply:
I would remove them :)  We have `STATUS.md` for knowing whats done. if later, such functions are indirectly tested by other tests, no one will take care of removing that functions. So it can be confusing i think.